stream/stream.h

```diff
@@ -228,6 +228,8 @@ struct bstr stream_read_complete(struct stream *s, void *talloc_ctx,
                                  int max_size);
 struct bstr stream_read_file(const char *filename, void *talloc_ctx,
                              struct mpv_global *global, int max_size);
+struct bstr stream_read_file2(const char *filename, void *talloc_ctx,
```

Comment:
Could we have short docs describing different purposes on those functions? So we don't have to look at source? I understand one doesn't take flags.

Reply:
Thanks.